probes/hasBinaryArtifacts/def.yml

```diff
@@ -15,12 +15,12 @@
 id: hasBinaryArtifacts
 short: Checks if the project has any binary files in its source tree.
 motivation: >
-  Binary files are not readable so users can't see what they do. Many programming language systems can generate executables from source code (e.g., C/C++ generated machine code, Java .class files, Python .pyc files, and minified JavaScript). Users will often directly use executables if they are included in the source repository, leading to many dangerous behaviors.
+  Binary files are not human readable so users and reviewers can't easily see what they do.
 implementation: >
   The implementation looks for the presence of binary files. This is a more restrictive probe than "hasUnverifiedBinaryArtifacts" which excludes verified binary files.
 outcome:
-  - If the probe finds binary files, it returns a number of true outcomes equal to the number of binary files found. Each outcome includes a location of the file.
-  - If the probe finds no binary files, it returns a single false outcome.
+  - If the probe finds binary files, it returns one OutcomeTrue for each binary file found.
```

Comment:
should `OutcomeTrue` be simply `True`? Same for `OutcomeFalse` should be `False`? The `onOutcome` field is set to `True`, so I think we should re-use the same value?

Reply:
so the constant is called `finding.OutcomeTrue`, but the value is `True` because otherwise the output would be:

```json
"outcome": "OutcomeTrue"
```

We can address this later if you want, but going to merge for now

Reply:
sure. What the constant is called in our code is an implementation detail imo. What users see is the yml and the json probe result, which both say `True`. Hence why I'm proposing using `True` in the yml.